.travis.yml

```diff
@@ -4,6 +4,14 @@ rust:
 branches:
   only:
     - bench
+before_install:
+  - curl -L https://downloads.sourceforge.net/project/gnuplot/gnuplot/5.0.6/gnuplot-5.0.6.tar.gz > gnuplot-5.0.6.tar.gz
+  - tar xzf gnuplot-5.0.6.tar.gz
+  - cd gnuplot-5.0.6
+  - ./configure
+  - make && make install
```

Comment:
Alright so there is a `gnuplot` package, but it isn't gnuplot 5.0.5 which is what I need for some features. So not entirely sure.